### DISCUSSION TITLE
Hello World Example?  Where is Doco

### DISCUSSION THREAD

**Discussion by dbareis** [brianvoe/slim-select#540] [Q&A] [answered]
"https://slimselectjs.com/selects#single" makes it look simple to use yet it doesn't work.  Where is documentation?  Is there a hello world example?  

While Googling I found at least one framework that moved off slim select to tom select mainly due to the lack of documentation.  I'm happy to use slim select if it is indeed as simple as adding a "new SlimSelect" and providing the ID of the select statement.  But that is not working here whether I place the JS before or after the creation of the select control created with PHP.

**Comment by dbareis** [comment, marked as answer]
OK I got it to work, here is my example (not sure what is wrong in my real code but I'll sort that out next :-(

**I'd still like to know where the doco is...**

```
<script src="JS Libraries/slim-select/v2.8.2/slimselect.min.js"></script>
<link rel="stylesheet" href="JS Libraries/slim-select/v2.8.2/slimselect.css">


<h1>Slim Select Test</h1>
<form id="upload_form" action="#" enctype="multipart/form-data" method="get">
   <select id='Db24Select' class="ignore">
        <option value="1">Some option</option>
        <option value="2">Another option</option>
        <option value="3" disabled="">A disabled option</option>
        <option value="4">Potato</option>
      </select>
</form>

<script>
    debugger;
    new SlimSelect({ select: '#Db24Select',
                        closeOnSelect: true,
                        allowDeselectOption: true,
                        showSearch: true,
                        selectByGroup: true,
    });
</script>
```

**Comment by dbareis** [comment]
Ok, so there is no doco, not a good idea, wonder how many people have gone to tom select or nice select or similar because of the lack of a simple standalone example.

**Comment by brianvoe** [comment]
Ya your right. Sorry this is the little documentation i have. 

Install
https://slimselectjs.com/install#npm
https://slimselectjs.com/install#cdn
https://slimselectjs.com/install#download

Simple
https://slimselectjs.com/selects#single
https://slimselectjs.com/selects#multiple

Data Usage
https://slimselectjs.com/data#types
https://slimselectjs.com/data#field

Settings
https://slimselectjs.com/settings#select
https://slimselectjs.com/settings#alwaysOpen
https://slimselectjs.com/settings#contentLocation
https://slimselectjs.com/settings#contentPosition
https://slimselectjs.com/settings#openPosition
https://slimselectjs.com/settings#placeholder
https://slimselectjs.com/settings#selectAll
https://slimselectjs.com/settings#allowDeselect
https://slimselectjs.com/settings#display
https://slimselectjs.com/settings#disabled
https://slimselectjs.com/settings#mandatory
https://slimselectjs.com/settings#minmax
https://slimselectjs.com/settings#dataAttributes
https://slimselectjs.com/settings#cssClass
https://slimselectjs.com/settings#inlineStyles
https://slimselectjs.com/settings#html
https://slimselectjs.com/settings#keepOrder
https://slimselectjs.com/settings#search
https://slimselectjs.com/settings#closeOnSelect
https://slimselectjs.com/settings#showOptionTooltips
https://slimselectjs.com/settings#closable
https://slimselectjs.com/settings#hideSelected
https://slimselectjs.com/settings#maxValuesShown

Events
https://slimselectjs.com/events#error
https://slimselectjs.com/events#beforeChange
https://slimselectjs.com/events#afterChange
https://slimselectjs.com/events#open
https://slimselectjs.com/events#search
https://slimselectjs.com/events#searchFilter
https://slimselectjs.com/events#addable

Methods
https://slimselectjs.com/methods#getSelected
https://slimselectjs.com/methods#setSelected
https://slimselectjs.com/methods#getData
https://slimselectjs.com/methods#setData
https://slimselectjs.com/methods#enableDisable
https://slimselectjs.com/methods#openClose
https://slimselectjs.com/methods#search
https://slimselectjs.com/methods#destroy

Frameworks
https://slimselectjs.com/vue
https://slimselectjs.com/react
